File: nathan/Problema0137/problema0137.py
dia = int(input("Digite o dia da data a ser inserida: "))
mes = int(input("Digite o mes da data a ser inserida: "))
ano = int(input("Digite o ano da data a ser inserida: "))

validarAno = False

if dia > 0 and dia <= 31:
    if mes > 0 and mes <= 12:
        if ano > 0:
            validarAno = True

print("Data inserida é valido? ", validarAno)
        

File: nathan/Problema0137/problema0137.java
package Problema0137;

import java.util.Scanner;
public class problema0137 {
    
    public static void main(String[] args){
    Scanner input = new Scanner(System.in);

    System.out.println("Digite o dia da data a ser inserida: ");
    int dia = Integer.valueOf(input.nextLine());
    System.out.println("Digite o mes da data a ser inserida: ");
    int mes = Integer.valueOf(input.nextLine());
    System.out.println("Digite o ano da data a ser inserida: ");
    int ano = Integer.valueOf(input.nextLine());

    boolean validarAno = false;

    if(dia > 0 && dia <= 31){
        if(mes > 0 && mes <= 12){
            if(ano > 0){
                validarAno = true;
            }
        }
    }

    System.out.println("Data inserida é valido? " + validarAno);
    }
}
